ui/src/overridden/backoffice/Items/ItemCirculationShelf.js

```diff
@@ -1,10 +1,11 @@
 import React from "react";
-import { Grid, Embed, Button, Icon } from "semantic-ui-react";
+import { Grid, Embed, Button, Icon, Message, List } from "semantic-ui-react";
 import PropTypes from "prop-types";
+import _isEmpty from "lodash/isEmpty";
 import { shelfLink } from "../../utils";
 
 export const ItemCirculationShelf = ({ metadata }) => {
-  return (
+  return !_isEmpty(metadata.shelf) ? (
```

Comment:
minor: ternary operators are quite unreadable. It would be easier to use return
```suggestion
  if (!_isEmpty(metadata.shelf)){
     return <Grid.Column width={6}> 
     ... 
     }
  return <Grid.Column ....
```